fe/fe-core/src/main/java/org/apache/doris/nereids/properties/PhysicalProperties.java

```diff
@@ -17,9 +17,20 @@
 
 package org.apache.doris.nereids.properties;
 
+import org.apache.doris.analysis.DistributionDesc;
```

Comment:
config `style` according to [doc](https://doris.apache.org/zh-CN/developer-guide/java-format-code.html#import-order)

Reply:
done